feature/security/gnsi/acctz/AccountingAuthenErrorMulti/README.md

```diff
@@ -0,0 +1,54 @@
+# gNSI.acctz.v1 (Accounting) Test Accounting Authentication Error - Multi-transaction
+
+Test Accounting for authentication errors in the authentication service
+for multi-transaction logins.  For example, unreachable TACACS+ server(s).
+
+## Procedure
+
+- Record the current time, T0
+- For one of the possible RecordResponse.service_request.service_type that is not authenticated per-transaction:
+	- Connect to the DUT, recording the local and remote IP addresses and port numbers,
+	- Provide the valid user credentials configured in [Accounting Authentication Error - Config](../AccountingAuthenErrorConfig/)
+	- disconnect
+- Establish gNSI connection to the DUT.
+- Call gnsi.acctz.v1.Acctz.RecordSubscribe with RecordRequest.timestamp = T0
+- Verify that accurate accounting records are returned for the commands/RPCs authentication failures.
+- If start/stop accounting was enabled, each authentication failure should have an accompanying LOGIN accounting record.
+- For the RecordResponse correlated to the connection made above, check/confirm that:
+	- session_info. :
```

Comment:
In feature/security/gnsi/acctz/RecordSubscribeNongrpc/README.md, I have several comments about the mentioned message and fields, which also apply here. Can you help update per the comment discussion? Thanks.